net.biniou.LeBiniou.yml

```diff
@@ -0,0 +1,101 @@
+        sha256: e8f445e85faf727b89e9f9590daea4473ae00ead38b237cf1eda55172b89b182
+    build-commands:
+      - ./configure --prefix=/app
+      - make -j4 install
+  - name: orcania
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/babelouest/orcania/archive/v2.1.1.tar.gz
+        sha256: 960074f6aa198e9db5440347d20cff8be3b82ee1c15ac7b23c0ac54a301a85cd
+    build-commands:
+      - make -j4 DESTDIR=/app install
+  - name: yder
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/babelouest/yder/archive/v1.4.12.tar.gz
+        sha256: 59bed6df43c29bf1a0f85274369f7a8bc7dfe5e94be826139e8aa05eda3ba2e6
+    build-commands:
+      - make -j4 DESTDIR=/app install
+  - name: jansson
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/akheron/jansson/archive/v2.13.1.tar.gz
+        sha256: f22901582138e3203959c9257cf83eba9929ac41d7be4a42557213a22ebcc7a0
+    build-commands:
+      - autoreconf -i
+      - ./configure --prefix=/app
+      - make -j4 install
+  - name: ulfius
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/babelouest/ulfius/archive/v2.7.1.tar.gz
+        sha256: d5dfb90ac16fe9d8ce70fe6b23e43102d5208d4f0174196d3ef183d950d3a57b
+    build-commands:
+      - make DESTDIR=/app UWSCFLAG=1 install
+  - name: imagemagick
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/ImageMagick/ImageMagick6/archive/6.9.11-51.tar.gz
+        sha256: 06c4363486e1053e69bb3fa211374bdaed4baa678ca8ccd1f38731f0da2d3497
+    build-commands:
+      - ./configure --prefix=/app
+      - make -j4 install
+  - name: jackaudio
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/jackaudio/jack2/archive/v1.9.16.tar.gz
+        sha256: e176d04de94dcaa3f9d32ca1825091e1b938783a78c84e7466abd06af7637d37
+    build-commands:
+      - ./waf configure --prefix=/app
+      - ./waf
+      - ./waf install
+  - name: lebiniou-data
+    buildsystem: simple
+    sources:
+       - type: git
+         url: https://gitlab.com/lebiniou/lebiniou-data.git
+         tag: version-3.50
+         commit: f4b03c90b838c4b84ab2cd56348dedf724e653b2
+    build-commands:
+      - autoreconf -i
+      - ./configure --prefix=/app
+      - make -j4 install
+  - name: lebiniou
+    buildsystem: simple
+    sources:
+       - type: git
+         url: https://gitlab.com/lebiniou/lebiniou.git
+         tag: version-3.52.0
+         commit: 02fc30edad6ef20396a8400104edb5bea539e308
+    build-commands:
+      - autoreconf -i
+      - ./configure --prefix=/app
+      - make -j4 install
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0
+  # used to send anonymous usage statistics
+  - --share=network
+  # users can put configuration and/or data files anywhere
+  - --filesystem=home
```

Comment:
Ultimately it's your decision, but I'd limit it to xdg-music and xdg-download. Users can change these permissions locally.

Reply:
I'll leave it as is for now.